src/CommonGLPI.php

```diff
@@ -1096,10 +1077,40 @@ class='btn btn-sm btn-icon btn-ghost-secondary'
                 "<i class='fa-lg ti ti-chevrons-right'></i></a>";
             }
 
+            echo "</div>";
+
             echo "</div>"; // .navigationheader
         }
     }
 
+    /**
+     * Compute the name to be used in the main header of this item
+     *
+     * @return string
+     */
+    public function getHeaderName(): string
+    {
+        $name = '';
+        if (isset($this->fields['id']) && ($this instanceof CommonDBTM)) {
+            $name = $this->getName();
+            if ($_SESSION['glpiis_ids_visible'] || empty($name)) {
+                $name = sprintf(__('%1$s - ID %2$d'), $name, $this->fields['id']);
+            }
+        }
+        if (
+            isset($this->fields["entities_id"])
+            && Session::isMultiEntitiesMode()
+            && $this->isEntityAssign()
+        ) {
+            $entname = Dropdown::getDropdownName("glpi_entities", $this->fields["entities_id"]);
+            if ($this->isRecursive()) {
+                $entname = sprintf(__('%1$s + %2$s'), $entname, __('Child entities'));
+            }
+            $name = sprintf(__('%1$s (%2$s)'), $name, $entname);
+        }
+
+        return $name;
```

Comment:
If item name contains `>`, `<` or `&` chars, they will be displayed as html entities. This method should return an unsanitized value, and twig will handle the escaping itself.

```suggestion
        return Sanitizer::unsanitize($name);
```

May require an additionnal use statement.